File: CodingSamples/DotNet/Fundamentals/Runtime/ReflectionTest/DemoApp1/Shop.cs
namespace DemoApp;

record Item(string Name, string Brand, int Stock = 125);

readonly record struct Customer(string Id, decimal Purchase, int Rating);

class Shop
{
    public static Item GetPopularItem()
    {
        return new Item("cpu", "intel");
    }

    public static Customer GetBestCustomer()
    {
        return new Customer("Jack", 150000, 5);
    }
}

File: CodingSamples/Java/Fundamentals/Runtime/ReflectionTest/demo1/Shop.java
record Item(String name, String brand, int stock){}

record Customer(String id, double purchase, int rating){}

class Shop {

    public static Item popularItem() {
        return new Item("cpu", "intel", 150);
    }

    public static Customer bestCustomer() {
        return new Customer("Jack", 150000, 5);
    }
}
